### DISCUSSION TITLE
Split band structure over multiple calculations

### DISCUSSION THREAD

**Discussion by sthartman** [band-unfolding/bandup#31] [Q&A]
Hello,
For large/expensive band structure calculations done non-selfconsistently, it's common practice to split the kpoints up among several independent calculations. Is it possible within BandUp to read a single band structure split across multiple output files?

**Comment by stepan-tsirkin** [comment]
No, it is not possible. But you may split yuor k-path, generate the PBZ points corresponding to each piece, do unfolding separately, and finally concatenate the results. I will think how to make it more convenient in  [banduppy](https://github.com/band-unfolding/banduppy)
